manager/controllers/app/data_catalog_interface.go

```diff
@@ -25,31 +23,33 @@ import (
 // - an error if happened
 // - the new asset identifier
 func (r *FybrikApplicationReconciler) RegisterAsset(catalogID string, info *app.DatasetDetails, input *app.FybrikApplication) (string, error) {
-	datasetDetails := &pb.DatasetDetails{}
-	err := info.Details.Into(datasetDetails)
-	if err != nil {
-		return "", err
-	}
+	/*	datasetDetails := &pb.DatasetDetails{}
```

Comment:
remove the commented code

Reply:
Are we sure about removing this ? There was an ask to support `registerAsset` after `getAssetInfo` is working. Do we want a reference in commented code here for that ? Please suggest. Thanks.

Reply:
When the time comes if you need this older code just pick at the git history ;)

Reply:
sure. fixed it now.